http-api-tests/broccoli-nomad-consul/api-v1-instances-service-status/expected/response-data

```diff
@@ -33,7 +33,7 @@
         }
       }
     },
-    "version": "9215ebc3b508fcc0295bffdbfe5695b2"
+    "version": "2c4c43cb791d1f4fda934c0e210d7c6d"
```

Comment:
Maybe this is correct but remind me please why the version changed.

Reply:
i don't know what those version numbers are about

Reply:
This is because the [new nomad needs non zero values for `Update.Stagger` and `Update.MaxParallel`](https://github.com/Data-Science-Platform/cluster-broccoli/pull/14/commits/94060e9af71ad89eb6ffcddaf4ba4057303905bb).